lib/index.js

```diff
@@ -76,8 +76,8 @@ api.localDocumentId = 'local';
 api.writeOptions = bedrock.config.mongodb.writeOptions;
 api.localWriteOptions = bedrock.config.mongodb.local.writeOptions;
 
-// load test config
-bedrock.events.on('bedrock.test.configure', () => require('./test.config'));
+-// load test config
+-bedrock.events.on('bedrock.test.configure', () => require('./test.config'));
```

Comment:
This looks like a failed merge or something. (erroneous minus signs)

Reply:
yeah I tried removing this, and consensus seems to be that we are not using `bedrock.test.configure` at the very least this seems to prove that hypothesis as the file was not being included I suppose. I would greatly like to do this in another issue so we can assure that all projects that depend on `bedrock-mongodb` via `bedrock-test` are still working once this has been removed. I removed the ticks so this is being required again. Original code: https://github.com/digitalbazaar/bedrock-mongodb/blob/211284e1245d9c07e5cedb87ef2c5276da9a2c54/lib/index.js#L79-L80

My commit: https://github.com/digitalbazaar/bedrock-mongodb/pull/44/commits/e711c5f54b8e47bb6871be403e9094de704a5154